heeji/모스부호(1).js

```diff
@@ -0,0 +1,10 @@
+const morse = {
+    '.-':'a','-...':'b','-.-.':'c','-..':'d','.':'e','..-.':'f',
+    '--.':'g','....':'h','..':'i','.---':'j','-.-':'k','.-..':'l',
+    '--':'m','-.':'n','---':'o','.--.':'p','--.-':'q','.-.':'r',
+    '...':'s','-':'t','..-':'u','...-':'v','.--':'w','-..-':'x',
+    '-.--':'y','--..':'z'
+}
+function solution(letter) {
+    return letter.split(' ').map(i => morse[i]).join('');
+}
```

Comment:
```suggestion
const morse = {
    '.-':'a','-...':'b','-.-.':'c','-..':'d','.':'e','..-.':'f',
    '--.':'g','....':'h','..':'i','.---':'j','-.-':'k','.-..':'l',
    '--':'m','-.':'n','---':'o','.--.':'p','--.-':'q','.-.':'r',
    '...':'s','-':'t','..-':'u','...-':'v','.--':'w','-..-':'x',
    '-.--':'y','--..':'z'
}
function solution(letter) {
    return letter.split(' ').map(item => morse[item]).join('');
}
```
- 그냥 i라고 쓰시면 `index`인지 `element`인지 알 수 없으니 명시적으로 적어주시는게 좋습니다!

Reply:
네! 피드백 감사합니다!ㅎ